src/olympia/lib/settings_base.py

```diff
@@ -551,52 +555,10 @@ def get_db_config(environ_var, atomic_requests=True):
     'https://addons.mozilla.org/api/v5/shelves/editorial'
 )
 
-# Filename where the strings will be stored. Used in puente config below.
+# Filename where the strings will be stored. Used in extract_content_strings
+# management command, but note that the filename is hardcoded in babel.cfg.
```

Comment:
it's a shame we can't define/generate all the babel config in one place 😞

Reply:
We could generate the babel config, but that's one more cog in the machine... Alternatively we could do what puente used to do and provide our own extract command, but I thought it'd be better if we tried to stay closer to how babel works...